docs/overview.yaml

```diff
@@ -424,17 +424,63 @@ Geometry:
     Nullstellensatz : 'MvPolynomial.vanishingIdeal_zeroLocus_eq_radical'
 
 Combinatorics:
-  Graph theory:
-    simple graph: 'SimpleGraph'
-    degree-sum formula: 'SimpleGraph.sum_degrees_eq_twice_card_edges'
-    matching: 'SimpleGraph.Subgraph.IsMatching'
-    adjacency matrix: 'SimpleGraph.adjMatrix'
   Pigeonhole principles:
     finite: 'Fintype.exists_ne_map_eq_of_card_lt'
     infinite: 'Finite.exists_infinite_fiber'
     strong pigeonhole principle: 'Fintype.exists_lt_card_fiber_of_mul_lt_card'
   Transversals:
     Hall's marriage theorem: 'Finset.all_card_le_biUnion_card_iff_exists_injective'
+  Enumerative combinatorics:
+    Catalan numbers: 'catalan'
+    Bell numbers: 'Multiset.bell'
+    Dyck words: 'DyckWord'
+  Set families:
+    Correlation inequalities:
+      Four functions theorem: 'four_functions_theorem'
+      Harris-Kleitman inequality: 'IsLowerSet.le_card_inter_finset'
+    Families of fixed size:
+      Kruskal-Katona theorem: 'Finset.kruskal_katona'
+      Ahlswede-Zhang identity: 'AhlswedeZhang.infSum_eq_one'
+      LYM inequality: 'Finset.sum_card_slice_div_choose_le_one'
+      Sperner's theorem: 'Finset.IsAntichain.sperner'
```

Comment:
What does `Families of fixed size` mean here?

Reply:
I mean $$X^(s)$$ for some set $$X$$ and natural $$s$$, namely all subsets of $$X$$ of size $$s$$. I realise now that the last three results aren't really about families of fixed size, but rather about the hypercube stratified by size. Do you have a suggestion for the name of this subsection?

Reply:
Maybe these four don't belong in a subsection - they can just come under set family combinatorics.

Reply:
The last three can be "Set antichains" maybe

Reply:
The last three can be "Set antichains" maybe

Reply:
I like this last suggestion. That's roughly what I originally was after, my motivation being that Sperner theory is a whole domain dedicated to studying antichains in posets